catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/adapter/BrickAdapter.kt

```diff
@@ -148,9 +148,18 @@ class BrickAdapter(private val sprite: Sprite) :
         item.checkBox.setOnClickListener { onCheckBoxClick(position) }
```

Comment:
please check if brick/checkbox is enabled before setting the onClickListener, else set an empty onClickListener, because the disabled brick shouldn't be clickable

```suggestion
        item.checkBox.setOnClickListener { onCheckBoxClick(position) }
        if (viewStateManager.isEnabled(position)) {
            itemView.setOnClickListener { onCheckBoxClick(position) }
        } else {
            itemView.setOnClickListener { }
        }
```